index.bs

```diff
@@ -329,6 +353,18 @@ A user agent implementing the Layout Instability API MUST include
 <a href="https://html.spec.whatwg.org/multipage/window-object.html#the-window-object">Window</a>
 contexts. This allows developers to detect support for the Layout Instability API.
 
+{{LayoutShiftAttribution}} interface {#sec-layout-shift-attribution}
+====================================================================
+
+<pre class="idl">
+  [Exposed=Window]
+  interface LayoutShiftAttribution {
+    readonly attribute Node node;
```

Comment:
Should this be nullable? And return null when the node is disconnected from the document